python-backend/app/api/mines/variances/models/variance.py

```diff
@@ -0,0 +1,66 @@
+from datetime import datetime
+
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.schema import FetchedValue
+
+from ....utils.models_mixins import AuditMixin, Base
+from app.extensions import db
+
+
+class Variance(AuditMixin, Base):
+    __tablename__ = "variance"
+    variance_id = db.Column(db.Integer, primary_key=True, server_default=FetchedValue())
+    compliance_article_id = db.Column(db.Integer,
+                                      db.ForeignKey('compliance_article.compliance_article_id'),
+                                      server_default=FetchedValue())
+    mine_guid = db.Column(UUID(as_uuid=True), db.ForeignKey('mine.mine_guid'))
+    note = db.Column(db.String(300), server_default=FetchedValue())
+    issue_date = db.Column(db.DateTime, nullable=False, default=datetime.utcnow)
+    received_date = db.Column(db.DateTime, nullable=False, default=datetime.utcnow)
+    expiry_date = db.Column(db.DateTime,
+                            nullable=False,
+                            default=datetime.strptime('9999-12-31', '%Y-%m-%d'))
+
```

Comment:
we should be able to set up a relationship that looks like this. 

`mine_documents = db.relationship("MineDocument", secondary='variance_document_xref')`

then the python just manipulates the list and sqlalchemy handles the join table.

Reply:
Okay. I still haven't warmed up to SQLAlchemy as much as I should. Appreciate you sharing these insights